apps/nuxt3-ssr/app.vue

```diff
@@ -9,7 +9,8 @@
     </div>
     <div class="z-30 relative min-h-screen flex flex-col">
       <slot name="header">
-        <AppHeader />
+        <HeaderCatalogue v-if="route.params.catalogue" />
+        <HeaderGlobal v-else />
```

Comment:
single header for consistency , maybe look at nuxt template feature to avoid huge complex app.vue file

Reply:
I see this as intermediate step; we need to discuss custom landing page of UMCG versus general ones. We might need some configuration for this.